organization/_posts/2018-05-16-packaging.md

```diff
@@ -0,0 +1,62 @@
+  - High reproducibility (up to 95% bitwise identical).
+  - Nix-style runtime env setup
+  - Everything lives under `/guix` (must be writable, user builds supported by daemon). Binary builds supported, but not relocatable (prefix enters package hash).
+  - User Base: mostly HPC, big shared filesystems user are close to (Graeme commented at seminar that typical HEP case is different; CVMFS etc)
+  - Good points like Nix: reproducibility, user control, composability (overlays)
+  - Differences: Scheme language vs Nix DSL (claimed Scheme makes use/extension easier). No mention of license
+  - Interesting for us? Yes, but lives very close to Nix. Limited resources, so not a priority unless someone wants to take it on. Maybe more important if we move in a Nix-like direction. Then differences between Nix DSL vs Scheme etc.
+  - Some observations: seems to have smaller package base than Nix, no apparent macOS distribution (but might build from source).
+- [CHEP timetable now available](https://indico.cern.ch/event/587955/timetable/#20180709)
+  - [Our packaging talk](https://indico.cern.ch/event/587955/sessions/266674/#20180711) followed by two Spack talks (one by Chris Green, one by IceCube)
+  - Guilherme: will have a [talk on robust binaries](https://indico.cern.ch/event/587955/contributions/2938043/) (RPATH, how linker finds libraries etc).
+- Today: Test Drive Round Table
+- SFT LIM Workshop in two weeks, so next meeting 13th June.
+- Just let Graeme and Ben know of any material you'd like to present.
+
+
+[Test Driving Round Table - Ben, all](https://indico.cern.ch/event/727088/contributions/2992556/attachments/1643140/2640160/HSF-TestStack.pdf)
+========================
+- Ben:
+  - [Github repo](https://github.com/HSF/packaging/tree/master/testdrive) has been updated with more instructions and guidance
+  - One directory per packager
+  - Overall README and a template for each specific tool
+  - Walk through the minimum packages and versions
+  - Demonstrate basic functionality first
+  - Template can be filled in so that each package manager is well described and easy to use by the non-expert test driver
+- Graeme: time limited on progress on Nix due to meetings and travel.
+- Chris Burr: With Nix, now packaging LHCb up to reconstruction packages.
+- Javier - has discussed with Patrick which repo to use for package descriptions. HSF looks the best. CERN one is very out of date.
+- Patrick has built most of the CMSSW stack, most updates went into HSF. Chris has been adding more versions, PRs going directly upstream into Spack. There is a FNAL fork that has useful changes to Spack itself and to some of the packages.
+  - Could add that information to packaging github repo, so that people could find it.
+  - Shazad has been working on a tool to convert SCRAM->CMake configurations, but it was taking too long for CMS (10 minutes - too slow!). CMake 3.11 is improved. Ninja target was faster than make. Some technical improvements being worked on.
```

Comment:
Suggest: "Shazad has been working on a tool to convert SCRAM->CMake configurations. However, resulting CMake configuration step was taking too long for CMS (10 minutes). CMake 3.11 improved the situation. Ninja target was faster than make. Some technical improvements being worked on."